observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,73 @@ These release notes track the development of the Network Observability Operator
 
+[id="network-observability-operator-release-notes-1-6_{context}"]
+== Network Observability Operator 1.6.0
+The following advisory is available for the Network Observability Operator 1.6.0:
+
+* link:https://access.redhat.com/errata/RHSA-2024:3868[Network Observability Operator 1.6.0]
+
+[id="network-observability-operator-1.6.0-features-enhancements_{context}"]
+=== New features and enhancements
+
+[id="network-observability-lokiless-enhancements-1.6_{context}"]
+==== Enhanced use of Network Observability Operator without Loki
+You can now use Prometheus metrics and rely less on Loki for storage when using the Network Observability Operator. For more information, see xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki].
+
+[id="network-observability-custom-metrics-1.6_{context}"]
+==== Custom metrics API
+You can create custom metrics out of flowlogs data by using the `FlowMetrics` API. Flowlogs data can be used with Prometheus labels to customize cluster information on your dashboards. You can add custom labels for any subnet that you want to identify in your flows and metrics. This enhancement can also be used to more easily identify external traffic by using the new labels `SrcSubnetLabel` and `DstSubnetLabel`, which exists both in flow logs and in metrics. Those fields are empty when there is external traffic, which gives a way to identify it. For more information, see xref:../../observability/network_observability/metrics-alerts-dashboards.adoc#network-observability-custom-metrics_metrics-dashboards-alerts[Custom metrics] and <FLOW_METRICS_API_REF_LINK_HERE>
+
+[id="network-observability-eBPF-performance-enhancements-1.6_{context}"]
+==== eBPF performance enhancements
+Experience improved performances of the eBPF agent, in terms of CPU and memory, with the following updates: 
+
+* The eBPF agent now uses TCX webhooks instead of TC.
+* The *NetObserv / Health* dashboard has a new section that shows eBPF metrics.
+** Based on the new eBPF metrics, an alert notifies you when the eBPF agent is dropping flows.
+* Loki storage demand decreases significantly now that duplicated flows are removed. Instead of having multiple, individual duplicated flows per network interface, there is one de-duplicated flow with a list of related network interfaces.
+
+[IMPORTANT]
+====
+With the duplicated flows update, the *Interface* and *Interface Direction* fields in the *Network Traffic* table are renamed to *Interfaces* and *Interface Directions*, so any bookmarked *Quick filter* queries using these fields need to be updated to `interfaces` and `ifdirections`. 
+====
+
+For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]. 
```

Comment:
Removed period here: 

```suggestion
For more information, see xref:../../observability/network_observability/network-observability-operator-monitoring.adoc#network-observability-netobserv-dashboard-ebpf-agent-alerts_network_observability[Using the eBPF agent alert]
```

and [Quick filters] actually links to Using the eBPF agent alert. Is that intended?

Reply:
For me, it goes to Filtering the network traffic, which is what I expect, since Quick filters is a header under that.